apps/src/templates/studioHomepages/UiTip.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const UiTip = React.createClass({
+  propTypes: {
+    index: React.PropTypes.number,
+    position: React.PropTypes.object,
+    text: React.PropTypes.string,
+    closeClicked: React.PropTypes.func.isRequired
+  },
+
+  closeClicked(index) {
+    this.props.closeClicked(index);
```

Comment:
as far as I can tell, the only possible value of `index` is `this.props.index`. So you are better off getting rid of the `index` parameter and just referencing `this.props.index` directly.